src/leo-client-app/src/components/FuturePasses.tsx

```diff
@@ -41,29 +43,35 @@ const FuturePasses: React.FC = () => {
   return (
     <div>
       <h1>Next Week's Passes</h1>
-      <table style={{ width: '100%', borderCollapse: 'collapse' }}>
+      <table style={{ width: "100%", borderCollapse: "collapse" }}>
         <thead>
           <tr>
-            <th style={{ width: '50%', border: '1px solid #000', padding: '8px' }}>Entry</th>
-            <th style={{ width: '50%', border: '1px solid #000', padding: '8px' }}>Exit</th>
+            <th
+              style={{
+                width: "50%",
+                border: "1px solid #000",
+                padding: "8px",
+              }}>
+              Entry
+            </th>
+            <th
+              style={{
+                width: "50%",
+                border: "1px solid #000",
+                padding: "8px",
+              }}>
+              Exit
+            </th>
           </tr>
         </thead>
         <tbody>
           {passes.map((passPair, index) => (
             <tr key={index}>
-              <td style={{ border: '1px solid #000', padding: '8px' }}>
-                {passPair[0].type === 'Enter' && (
-                  <>
-                    {passPair[0].time}
-                  </>
-                )}
+              <td style={{ border: "1px solid #000", padding: "8px" }}>
+                {passPair[0].type === "Enter" && <>{passPair[0].time}</>}
               </td>
-              <td style={{ border: '1px solid #000', padding: '8px' }}>
-                {passPair[1].type === 'Exit' && (
-                  <>
-                    {passPair[1].time}
-                  </>
-                )}
+              <td style={{ border: "1px solid #000", padding: "8px" }}>
+                {passPair[1].type === "Exit" && <>{passPair[1].time}</>}
               </td>
             </tr>
           ))}
```

Comment:
can you add a screenshot of what this styling is? i know it doesnt matter but hard to approve without seeing what it looks like